src/routes/+layout.svelte

```diff
@@ -2,11 +2,13 @@
 	import '@skeletonlabs/skeleton/themes/theme-crimson.css';
 	import '@skeletonlabs/skeleton/styles/all.css';
 	import '../app.postcss';
-	import { AppBar, AppShell, LightSwitch } from '@skeletonlabs/skeleton';
-	import { signOut } from '@auth/sveltekit/client';
+	import { AppBar, AppShell } from '@skeletonlabs/skeleton';
 	import { setLocale } from '$lib/i18n/i18n-svelte.js';
 	import type { LayoutData } from './$types.js';
 
+	// zeslint-disable-next-line import/no-named-as-default-member
+	import NavTrail from './NavTrail.svelte';
```

Comment:
WHAT???

Reply:
I had an error if I didn't remove this, was testing if I "removed" it by adding z it shouldn't look at it.